src/get-changelog.js

```diff
@@ -103,7 +103,6 @@ async function _getChangelog({
       preset: require('standard-version/lib/preset-loader')(defaults),
       tagPrefix,
       releaseCount,
-      pkg: { path: cwd },
```

Comment:
How did this fix it?

Reply:
> How did this fix it?

I was looking at args https://github.com/conventional-changelog/conventional-changelog/blob/master/packages/conventional-changelog-cli/cli.js#L36 and `pkg` seems to be redundant since we already specified path

Reply:
Did you see the JS version that we use? Sometimes the CLI doesn't work the same as the direct JS. A redundant field should be an issue?

Reply:
what I'm saying is `pkg` by default finds closest package.json. Otherwise, we have to include package.json in path
`pkg: { path: `${cwd}/package.json` },`

Reply:
I see. I think `pkg: { path: path.join(cwd, 'package.json') },` would be faster because it doesn't have to crawl.